flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/DefaultVertexParallelismAndInputInfosDecider.java

```diff
@@ -546,7 +553,7 @@ static DefaultVertexParallelismAndInputInfosDecider from(
                 configuration.get(
                         BatchExecutionOptions.ADAPTIVE_AUTO_PARALLELISM_AVG_DATA_VOLUME_PER_TASK),
                 configuration.get(
-                        BatchExecutionOptions
-                                .ADAPTIVE_AUTO_PARALLELISM_DEFAULT_SOURCE_PARALLELISM));
+                        BatchExecutionOptions.ADAPTIVE_AUTO_PARALLELISM_DEFAULT_SOURCE_PARALLELISM),
+                configuration.get(CoreOptions.DEFAULT_PARALLELISM));
     }
```

Comment:
parallelism.default should be obtained from ExecutionConfig instead of jobmaster Configuration

Reply:
> parallelism.default should be obtained from ExecutionConfig instead of jobmaster Configuration

Sorry for my mistake, I changed the code.

Reply:
> parallelism.default should be obtained from ExecutionConfig instead of jobmaster Configuration

Thank you very much for review. I want to get the parallelism of ExecutionConfig as the default value.
```java
  public int getParallelism() {
         return configuration.get(CoreOptions.DEFAULT_PARALLELISM);
  }
```